__tests__/send.test.js

```diff
@@ -24,7 +24,7 @@ test('Should be able to continue to next step', async (t) => {
 
   await t
     .click(
-      Selector('.Select')
+      Selector('[data-testid="selector"]')
```

Comment:
I can't seem to get past the first step in the send flow at all. Weird that this test passes.

(Not necessarily related to the place I'm placing the comment)

Reply:
Strange indeed. Fixed the flow by correctly setting the `value` attribute.